SPECS/rabbitmq-server/executeRabbitMQTests.sh

```
@@ -0,0 +1,122 @@
+#
+# The 3 expected failures are the following:
+# //deps/rabbitmq_peer_discovery_aws:integration_SUITE
+# //deps/rabbitmq_auth_backend_ldap:system_SUITE
+# //deps/rabbitmq_ampq1_0:system_SUITE - shard 1 of 2 specifically
+#
+# RabbitMQ depends on bazel 'latest'. As of 2/22/23, the latest bazel version is '6.0.0'.
+#
+# Some tests also rely on the 'mandoc' package. As of 2/22/23, this package was not available in Mariner. As a result, the package is
+# pulled from source and installed. These test break the entire run, so this dependency must be pulled in for the entire suite to succeed.
+#
+# The elixir package relies on having the UTF-8 character encoding set in your locale in order to function properly. Because of this, you 
+# set bazel to use the UTF-8 locale that you'd like with the --test-env flag at runtime, unless UTF-8 is the true system default (in this case
+# no additional flags are needed at runtime for elixir).
+#
+# Be sure to check the rabbitmq official repository for any testing updates before executing this script in case the process has changed in
+# any capacity or become more well documentated:
+#   README.md: https://github.com/rabbitmq/rabbitmq-server/blob/main/README.md
+#   BAZEL.md:  https://github.com/rabbitmq/rabbitmq-server/blob/main/BAZEL.md
+#
+#### END TESTING DISCLAIMER ####
+
+####   SCRIPT ASSUMPTIONS   ####
+# The following line has been added to `$RABBIT_MQ_DIR/.bazelrc`:
+#   build --cache_test_results=no
+#
+# `$RABBIT_MQ_DIR/user.bazelrc` has been created and updated from the `user-template.bazelrc` in the rabbitmq repo.
+# 
+# This script should be run with `sudo` or with similar root access to directories like /usr/local/bin and /usr/local/sbin
+#
+# The PARENT_DIR variable must be set to the parent directory of your rabbitmq directory.
+#### END SCRIPT ASSUMPTIONS ####
+
+# Setup Path Variables
+PARENT_DIR="/path/to/rabbitmq/parent/directory" # <------ SET ME!!!!!
```

Maybe take this as an argument instead?  Fail if not set?